### PR TITLE
chore(egen): follows new template, fixes typos, K80-->T4, REGION-->LOCATION etc.

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR makes the following updates to the `multicontender_vs_champion_deloyment_method.ipynb` notebook:

- Change REGION to LOCATION
- Remove IS_TESTING condition at cleaning step
- Add Colab enterprise link
- Add Colab specific steps and other updates according to the new notebook template
- Remove future tense
- Contract words (like do not --> don't etc.)
- Fix the position of a sub-section (for "Save the model artifacts")
- Replace K80 with T4 in the text about GPUs
- Fix the typo "modesl"
- Moves the clean up step for yaml file to the last section
- Removes unnecessary code highlights (using "``")
- Handles the cases for Vertex AI resources
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 1]
```diff
@@ -633,7 +552,7 @@
       "source": [
         "## Get pretrained model from TensorFlow Hub\n",
         "\n",
-        "For demonstration purposes, this tutorial uses a pretrained model from TensorFlow Hub (TFHub), which is then uploaded to a `Vertex AI Model` resource. Once you have a `Vertex AI Model` resource, the model can be deployed to a `Vertex AI Endpoint` resource.\n",
+        "For demonstration purposes, this tutorial uses a pretrained model from TensorFlow Hub (TFHub), which is then uploaded to a Vertex AI model resource. Once you have Vertex AI model resource, the model can be deployed to a Vertex AI endpoint resource.\n",
```
Comment (katiemn) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:555]: Keep the 'a' in this sentence

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 2]
```diff
@@ -657,6 +576,17 @@
...
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dcee58290f19"
+      },
+      "source": [
+        "### Save the model artifacts\n",
+        "\n",
+        "At this point, the model is in memory. Next, you save the model artifacts to a Cloud Storage location."
```
Comment (katiemn) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:587]: Can remove 'you' here

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 3]
```diff
@@ -818,21 +749,21 @@
...
         "\n",
         "In this example, you deploy the model with the minimal amount of specified parameters, as follows:\n",
         "\n",
-        "- `model`: The `Model` resource.\n",
+        "- `model`: The model resource.\n",
         "- `deployed_model_displayed_name`: The human readable name for the deployed model instance.\n",
         "- `machine_type`: The machine type for each VM instance.\n",
         "\n",
-        "Do to the requirements to provision the resource, this may take upto a few minutes."
+        "This may take upto a few minutes due to the provisioning of the configured resources."
```
Comment (katiemn) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:766]: Can remove 'upto' here

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 4]
```diff
@@ -585,32 +513,21 @@
...
-        " - `vCPUs`: number of \\[2, 4, 8, 16, 32, 64, 96 \\]\n",
+        "- Set the variable `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for deployment. For this, you specify:\n",
+        "     - `machine_type`\n",
+        "         - `n1-standard`: 3.75GB of memory per vCPU.\n",
+        "         - `n1-highmem`: 6.5GB of memory per vCPU\n",
+        "         - `n1-highcpu`: 0.9 GB of memory per vCPU\n",
+        "     - `vCPUs`: number of \\[2, 4, 8, 16, 32, 64, 96 \\]\n",
         "\n",
-        "*Note: The following is not supported for training:*\n",
+        "**Note**: The following are not supported.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:525]: change "are not" to "aren't"

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 5]
```diff
@@ -675,17 +606,17 @@
...
-        "- `display_name`: A human readable name for the `Model` resource.\n",
+        "- `display_name`: A human readable name for the model resource.\n",
         "- `artifact_uri`: The Cloud Storage location of the model package.\n",
         "- `serving_container_image_uri`: The serving container image.\n",
         "\n",
-        "Uploading a model into a Vertex AI Model resource returns a long running operation, since it may take a few moments. \n",
+        "Uploading a model into a Vertex AI Model Registry returns a long running operation, since it may take a few moments.\n",
         "\n",
-        "*Note:* When you upload the model artifacts to a `Vertex AI Model` resource, you specify the corresponding deployment container image."
+        "**Note:** When you upload the model artifacts to Vertex AI Model Registry, you specify the corresponding deployment container image."
```
Comment (kittyabs) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:619]: delete "you"

-  "When you upload the model artifacts to Vertex AI Model Registry, specify the corresponding deployment container image"

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 6]
```diff
@@ -757,7 +688,7 @@
         "\n",
         "Next, upload the model's evaluation from the custom training job to the corresponding entry in the Vertex AI Model Registry.\n",
         "\n",
-        "Currently, there is not yet support for this method in the SDK. Instead, you use the lower level GAPIC API interface."
+        "Currently, there is no support for this method yet in the Python SDK. Instead, you use the lower level GAPIC API interface."
```
Comment (kittyabs) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:691]: suggested edit:

- Currently, the Python SDK does not support for this method. Instead, use the lower level GAPIC API interface.

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 7]
```diff
@@ -958,10 +889,10 @@
...
         "First, you define a component to import the evaluation metrics for the challenger model to the Model Registry. The component takes the following arguments:\n",
         "\n",
-        "- display_name: Human readable name for the evaluation metrics\n",
-        "- metrics: The evaluation metrics formatted for classification.\n",
-        "- parent_model_resource: The full resource name for the challenger model version.\n",
-        "- region: The region."
+        "- `display_name`: Human readable name for the evaluation metrics\n",
+        "- `metrics`: The evaluation metrics formatted for classification.\n",
+        "- `parent_model_resource`: The full resource name for the challenger model version.\n",
+        "- `region`: The region where you want to create/use the resources."
```
Comment (kittyabs) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:895]: remove / and change to:
- The region where you want to create or use the resources.

File: notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb [hunk 8]
```diff
@@ -1249,9 +1183,9 @@
       "source": [
         "### Get the latest state of the production endpoint\n",
         "\n",
-        "Now that the pipeline has finished, the contender (version 3) model has replaced the previous champion model on the production endpoint.\n",
+        "Once your pipeline execution is finished, you can notice that the contender (version 3) model has replaced the previous champion model on the production endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb:1186]: suggested edit:

-  "Once your pipeline execution is finished, notice that the contender (version 3) model has replaced the previous champion model on the production endpoint."

**Review by kittyabs — COMMENTED** [review]
some suggested edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**